src/promptflow/promptflow/azure/_restclient/flow_service_caller.py

```diff
@@ -134,7 +134,7 @@ def _get_user_identity_info(self):
         import jwt
 
         token = get_arm_token(credential=self._credential)
-        decoded_token = jwt.decode(token.token, options={"verify_signature": False})
+        decoded_token = jwt.decode(token, options={"verify_signature": False})
```

Comment:
Return the decoded token in `get_arm_token` to avoid decode again here?

Reply:
Temporarily not modified, the reason is:
1. Make the return value and semantics of the "**get_arm_token**" function clearer. Although there is an internal decode, it is only to verify the correctness of the token, and the parameters used during the decode are inconsistent with here
2. Decode currently performs very well, repeating it once has no significant impact.

Reply:
Maybe add some annotation on function's interface to mark the return value changed to string instead of token object?

Reply:
ok, the return type has been added.